includes/template-library/sources/local.php

```diff
@@ -622,20 +622,22 @@ public function get_item( $template_id ) {
 	 * @return array Local template data.
 	 */
 	public function get_data( array $args ) {
-		$db = Plugin::$instance->db;
-
 		$template_id = $args['template_id'];
 
-		// TODO: Validate the data (in JS too!).
-		if ( ! empty( $args['display'] ) ) {
-			$content = $db->get_builder( $template_id );
-		} else {
-			$document = Plugin::$instance->documents->get( $template_id );
-			$content = $document ? $document->get_elements_data() : [];
-		}
+		$document = Plugin::$instance->documents->get( $template_id );
+		$content = [];
 
-		if ( ! empty( $content ) ) {
-			$content = $this->replace_elements_ids( $content );
+		if ( $document ) {
```

Comment:
Tests?

- $args['display'] - is empty
- $args['display'] - not empty
- document with content
- document without content